pootle/apps/virtualfolder/models.py

```diff
@@ -54,8 +56,8 @@ class VirtualFolder(models.Model):
         help_text=_('Number specifying importance. Greater priority means it '
                     'is more important.'),
     )
-    is_browsable = models.BooleanField(
-        _('Is browsable?'),
+    is_public = models.BooleanField(
+        _('Is public?'),
         default=True,
         help_text=_('Whether this virtual folder is active or not.'),
```

Comment:
Based on a previous IRC discussion, I guess *active* sort of contradicts with *public* here, no?